src/ui/projects/CreateScenePage.js

```diff
@@ -0,0 +1,366 @@
+    [thumbnailUrl, setThumbnailUrl, setThumbnailFile]
+  );
+
+  const onPublish = useCallback(
+    function(e) {
+      e.preventDefault();
+
+      setError(null);
+      setIsUploading(true);
+
+      const abortController = new AbortController();
+
+      api
+        .publishGLBScene(
+          thumbnailFile,
+          glbFile,
+          {
+            name: sceneInfo.name,
+            allow_remixing: sceneInfo.allowRemixing,
+            allow_promotion: sceneInfo.allowPromotion,
+            attributions: {
+              creator: sceneInfo.creatorAttribution,
+              content: []
+            }
+          },
+          abortController.signal,
+          sceneId
+        )
+        .then(() => history.push("/projects"))
+        .catch(e => {
+          setIsUploading(false);
+          setError(e.message);
+        });
+    },
+    [thumbnailFile, glbFile, sceneInfo, sceneId, setIsUploading, api, history, setError]
+  );
+
+  const isNew = !sceneId;
+
+  const content = isLoading ? (
+    <ProgressBar />
+  ) : (
+    <>
+      <SceneUploadHeader>
+        <h1>{isNew ? "Publish Scene From GLB" : "Update GLB Scene"}</h1>
+
+        {sceneUrl && (
+          <Button disabled={isUploading} onClick={openScene}>
+            {configs.isMoz() ? "Open in Hubs" : "Open Scene"}
+          </Button>
+        )}
+      </SceneUploadHeader>
+      <SceneUploadFormContainer>
+        <LeftContent>
+          <label htmlFor="screenshotFile">
+            {thumbnailUrl ? <img src={thumbnailUrl} /> : <div>Click to select scene thumbnail (16:9 .png)</div>}
+          </label>
+          <input
+            id="screenshotFile"
+            type="file"
+            required={isNew}
+            accept=".png,image/png"
+            onChange={e => onChangeThumbnailFile(e.target.files)}
+          />
+        </LeftContent>
+
+        <RightContent>
+          <FormField>
+            <label htmlFor="sceneName">Scene Name</label>
+            <StringInput
+              id="sceneName"
+              required
+              pattern={"[A-Za-z0-9-':\"!@#$%^&*(),.?~ ]{4,64}"}
+              title="Name must be between 4 and 64 characters and cannot contain underscores"
+              value={sceneInfo.name}
+              onChange={onChangeName}
+            />
+          </FormField>
+          <FormField>
+            <label htmlFor="creatorAttribution">Your Attribution (optional):</label>
+            <StringInput
+              id="creatorAttribution"
+              value={sceneInfo.creatorAttribution}
+              onChange={onChangeCreatorAttribution}
+            />
+          </FormField>
+          <FormField>
+            <FormField inline>
+              <label htmlFor="allowRemixing">
+                Allow{" "}
+                <a
+                  href="https://github.com/mozilla/Spoke/blob/master/REMIXING.md"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  Remixing
```

Comment:
We'll want to update this copy to reflect the changes @emclaren has proposed.

Reply:
Took a look around and wasn't able to find this copy. Since it hasn't been merged anyway it makes sense for this to be consistent with the publish dialog. We will just need to make sure to update this in the PR where we change the other publish flow copy.